### ISSUE TITLE
PROPOSAL: mirror the vc http api structure

### ISSUE THREAD

**Issue by OR13** [w3c-ccg/traceability-interop#8]
I suggest we mirror or directly link to the vc http api now that is has separate spec, and that we clearly separate endpoints not defined in the vc http api to avoid confusion.

**Comment by mprorock** [comment]
fully agree on this.  Do you want me to write a script to keep it in sync, or embed directly?

**Comment by OR13** [comment]
I think we can direct embed, CORs will probably allow it, lets start with that.

**Comment by mprorock** [comment]
> lets start with that.

works for me - we will give it a roll and see where it lands

**Comment by OR13** [comment]
Some work on this is already done, currently semi blocked by the vc http api holder api structure, defined here:

https://github.com/w3c-ccg/vc-http-api/issues/187

Ideally we would link directly to 2 sections 1 for internal and 1 for external.

**Comment by mprorock** [comment]
closing per weekly call
